blocks/rich-text/index.js

```diff
@@ -387,6 +400,19 @@ export class RichText extends Component {
 		this.context.onCreateUndoLevel();
 	}
 
+	getContainerNode() {
+		// Find the parent "relative" or "absolute" positioned container
+		const findRelativeParent = ( node ) => {
+			const style = window.getComputedStyle( node );
+			if ( style.position === 'relative' || style.position === 'absolute' ) {
```

Comment:
I'm wondering if we could do something like `!== 'static'` here. Totally optional, I know this is exciting code.

Reply:
That's a good idea! Not needed now though, since this function was removed in https://github.com/WordPress/gutenberg/commit/45221727c4a4283637968792f6a38700697f1a9f.